clarified /etc/passwd requirement when running images for CRI-O compa… (#64572)
openshift/openshift-docs: modules/images-create-guide-openshift.adoc

[WARNING]
====

> Care must be taken when altering the directories and file permissions of sensitive areas of a container, which is no different than to a normal system. If applied to sensitive areas, such as `/etc/passwd`, this can allow the modification of such files by unintended users potentially exposing the container or host. CRI-O supports the insertion of arbitrary user IDs into the container's `/etc/passwd`, so changing permissions is never required.

Reviewer: I know this is existing text. Can you: 
```suggestion
Care must be taken when altering the directories and file permissions of the sensitive areas of a container. If applied to sensitive areas, such as `/etc/passwd`, such changes can allow the modification of these files by unintended users, potentially exposing the container or host. CRI-O supports the insertion of arbitrary user IDs into a container's `/etc/passwd`, so changing permissions is never required.
```

> Additionally, the /etc/passwd file should not exist in the container image. If it does, the CRI-O container runtime will fail to inject a random UID into /etc/passwd. In such cases, the container might face challenges in resolving the active UID. Failing to meet this requirement could impact the functionality of certain containerized applications.

Reviewer: ```suggestion
Additionally, the `/etc/passwd` file should not exist in any container image. If it does, the CRI-O container runtime will fail to inject a random UID into the `/etc/passwd` file. In such cases, the container might face challenges in resolving the active UID. Failing to meet this requirement could impact the functionality of certain containerized applications.
```
